src/lib/gml-parser/fuzz/fuzz_targets/gml_parse.rs

```diff
@@ -0,0 +1,6 @@
+#![no_main]
```

Comment:
This doesn't look to be a part of shadow.

Reply:
it's included in the main crate [here](https://github.com/shadow/shadow/blob/98f0217a13e643a4f7ca56fb66203be6d3c45031/src/main/Cargo.toml#L22) and used [here](https://github.com/shadow/shadow/blob/98f0217a13e643a4f7ca56fb66203be6d3c45031/src/main/network/network_graph.rs#L31) and elsewhere in that file